entrypoint_debian.sh

```diff
@@ -1,46 +1,27 @@
 #!/bin/bash -x
-
-# Karaf needs a pseudo-TTY so exit and instruct user to allocate one when necessary
-test -t 0
-if [ $? -eq 1 ]; then
-    echo "Please start the openHAB container with a pseudo-TTY using the -t option or 'tty: true' with docker compose"
-    exit 1
-fi
-
```

Comment:
Why remove these lines?

Reply:
I didn't intentionally remove any lines. Not sure what caused that. I'll look into it. My changes should only include additions and all in the same place,